### PR TITLE
What's new in 3.8: Fix some typos

### PR DESCRIPTION
**Pull request by hugovk**
Please add the `skip issue` and `skip news` labels.

### CODE REVIEW

**Comment by tirkarthi** [comment]
See also #16621

**Comment by tirkarthi** [comment]
Sorry, just noticed you reviewed the PR.

File: Doc/whatsnew/3.8.rst [hunk 1]
```diff
@@ -215,7 +215,7 @@ subdirectories).
 Debug build uses the same ABI as release build
 -----------------------------------------------
 
-Python now uses the same ABI whether it built in release or debug mode. On
+Python now uses the same ABI whether it's built in release or debug mode. On
```
Comment (CuriousLearner) [inline Doc/whatsnew/3.8.rst:218]: I think this should be:

```suggestion
Python now uses the same ABI whether it's built-in release or debug mode. On
```

Comment (hugovk) [reply Doc/whatsnew/3.8.rst:218]: I think the original is correct. 

Compare: "Python is built in release mode."

**Pull request merged**